README.md

```diff
@@ -1,14 +1,28 @@
 Algolia Search for Magento 2
 ==================
 
-![Latest version](https://img.shields.io/badge/latest-1.13.2-green.svg)
+![Latest version](https://img.shields.io/badge/latest-1.13.3-green.svg)
 ![Magento 2](https://img.shields.io/badge/Magento-%3E=2.2%20<2.4-blue.svg)
 ![PHP >= 7.0.6](https://img.shields.io/badge/PHP-%3E=7.0-green.svg)
 
 [![CircleCI](https://circleci.com/gh/algolia/algoliasearch-magento-2/tree/master.svg?style=svg)](https://circleci.com/gh/algolia/algoliasearch-magento-2/tree/master)
 
 -------
 
+#### Magento 2.4 compatibility & extension's versions End of Life
+
+We are happy to announce that the version 3.x of our extension is now compatible with Magento 2.4 (see details below in this readme to know more about differences between the different branches).
+
+We also decided to stop maintaining actively legacy branches (1.x and 2.x), we'll just continue to backport major fixes and we'll release the latest version of each of these branches in January 2021. We won't accept any PRs after this date.
+
+| Extension Version | End of Life |
+| --- | --- |
+| v1.x | Jan 2021 |
+| v2.x | Jan 2021 |
+| v3.x | N/A |
+
+-------
```

Comment:
Can we move this announcement below support one? That should always be at the top!